data/json/recipes/food/seed.json

```diff
@@ -1,5 +1,161 @@
 [
     {
+        "result": "seed_pumpkin",
+        "type": "recipe",
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_OTHER",
+        "skill_used": "survival",
+        "difficulty": 2,
+        "time": 5000,
+        "autolearn": true,
+        "qualities": [ { "id": "CUT", "level": 1 } ],
+        "components": [ [ [ "pumpkin", 1 ] ] ]
+    },
+    {
+        "result": "seed_canola",
+        "type": "recipe",
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_OTHER",
+        "skill_used": "survival",
+        "difficulty": 2,
+        "time": 5000,
+        "autolearn": true,
+        "qualities": [ { "id": "CUT", "level": 1 } ],
+        "components": [ [ [ "canola", 1 ] ] ]
+    },
+    {
+        "result": "seed_cucumber",
+        "type": "recipe",
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_OTHER",
+        "skill_used": "survival",
+        "difficulty": 2,
+        "time": 5000,
+        "autolearn": true,
+        "qualities": [ { "id": "CUT", "level": 1 } ],
+        "components": [ [ [ "cucumber", 1 ] ] ]
+    },
+    {
+        "result": "seed_potato_raw",
+        "type": "recipe",
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_OTHER",
+        "skill_used": "survival",
+        "difficulty": 2,
+        "time": 5000,
+        "autolearn": true,
+        "qualities": [ { "id": "CUT", "level": 1 } ],
+        "components": [ [ [ "potato_raw", 1 ] ] ]
+    },
+    {
+        "result": "seed_oats",
+        "type": "recipe",
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_OTHER",
+        "skill_used": "survival",
+        "difficulty": 2,
+        "time": 5000,
+        "autolearn": true,
+        "qualities": [ { "id": "CUT", "level": 1 } ],
+        "components": [ [ [ "oats", 1 ] ] ]
+    },
+    {
+        "result": "marloss_seed",
+        "type": "recipe",
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_OTHER",
+        "skill_used": "survival",
+        "difficulty": 2,
+        "time": 5000,
+        "autolearn": true,
+        "qualities": [ { "id": "CUT", "level": 1 } ],
+        "components": [ [ [ "marloss_berry", 1 ] ] ]
+    },
+    {
+        "result": "seed_corn",
+        "type": "recipe",
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_OTHER",
+        "skill_used": "survival",
+        "difficulty": 2,
+        "time": 5000,
+        "autolearn": true,
+        "qualities": [ { "id": "CUT", "level": 1 } ],
+        "components": [ [ [ "corn", 1 ] ] ]
```

Comment:
The item id for canned corn is can_corn, "corn" is an ear of corn or corn kernels